feito/repository.py

```diff
@@ -0,0 +1,21 @@
+from git import Repo
+import subprocess
+
+
+class Repository:
+
+    def __init__(self):
+        self.repo_name = self.repo_name()
+        self.last_commit_id = self.commit_id()
+
+    @staticmethod
+    def repo_name():
+        repo_name = subprocess.run(
+          "basename `git rev-parse --show-toplevel`", shell=True, stdout=subprocess.PIPE
+        ).stdout
```

Comment:
pylint: An attribute defined in feito.repository line 11 hides this method. Code: method-hidden